### PR TITLE
Create appointed.js

### PR DESCRIPTION
**Pull request by DukeTax**
Implement appointed. Hope I'm not making a fool of myself. :P

### CODE REVIEW

File: server/game/cards/attachments/05/appointed.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Appointed extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: [
+            ability.effects.addIcon('intrigue'),
```
Comment (cryogen) [inline server/game/cards/attachments/05/appointed.js]: please add another level of indenting here

**Comment by cryogen** [comment]
Implementation looks good, but please rename the file to appointed.jsx :)

**Comment by cryogen** [comment]
Forget what I said, I have been playing AGoT too much today.  Will merge as-is :)  Thanks!

**Comment by DukeTax** [comment]
That's fine. Was still trying to find my way around github anyway. :)

**Pull request merged**